store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/integration/FederatedAdminIT.java

```diff
@@ -581,11 +579,11 @@ public void shouldChangeGraphIdInStorage() throws Exception {
                 .build(), user);
 
         //then
-        ArrayList<String> graphIds = Lists.newArrayList(graph.execute(new GetAllGraphIds(), user));
+        Iterable<String> graphIds = (Iterable<String>) graph.execute(new GetAllGraphIds(), user);
```

Comment:
You could use the following:
```
Iterable<?> graphIds = graph.execute(new GetAllGraphIds(), user);

assertThat(changed).isTrue();
assertThat(graphIds).asInstanceOf(InstanceOfAssertFactories.iterable(String.class))
        .hasSize(1)
        .containsExactly(newName);
```
It means you do not have to suppress warnings.

Reply:
The point is you are letting another class deal with it. If you don't want to implement this then at least add a `@SuppressWarnings` annotation on the method